server/impl/src/main/java/com/walmartlabs/concord/server/org/secret/SecretManager.java

```diff
@@ -274,6 +279,16 @@ public DecryptedBinaryData createBinaryData(DSLContext tx,
         return new DecryptedBinaryData(id);
     }
 
+    public ApiKeyEntry assertApiKey(AccessScope accessScope, UUID orgId, String secretName, String password) {
+        DecryptedSecret secret = getSecret(accessScope, orgId, secretName, password, SecretType.DATA);
+        BinaryDataSecret data = (BinaryDataSecret)secret.getSecret();
+        ApiKeyEntry result = apiKeyDao.find(new String(data.getData()));
+        if (result == null) {
+            throw new ConcordApplicationException("Api key from secter '" + secretName + "' not found", Status.NOT_FOUND);
```

Comment:
Hate to point this out, misspelled `secter` -> `secret`

Reply:
ouch

Reply:
fixed